package.json

```diff
@@ -35,29 +36,37 @@
   "author": "Ernesto Garcia <gnapse@gmail.com> (http://gnapse.github.io/)",
   "license": "MIT",
   "dependencies": {
-    "@babel/runtime": "^7.5.1",
+    "@babel/runtime": "^7.8.3",
     "@types/testing-library__jest-dom": "^5.0.0",
-    "chalk": "^2.4.1",
-    "css": "^2.2.3",
+    "chalk": "^3.0.0",
+    "css": "^2.2.4",
     "css.escape": "^1.5.1",
-    "jest-diff": "^24.0.0",
-    "jest-matcher-utils": "^24.0.0",
-    "lodash": "^4.17.11",
-    "pretty-format": "^24.0.0",
+    "jest-diff": "^25.1.0",
+    "jest-matcher-utils": "^25.1.0",
+    "lodash": "^4.17.15",
+    "pretty-format": "^25.1.0",
     "redent": "^3.0.0"
   },
   "devDependencies": {
-    "jest-watch-select-projects": "^0.1.2",
-    "jsdom": "^15.1.0",
-    "kcd-scripts": "^1.4.0"
+    "jest-environment-jsdom-sixteen": "^1.0.0",
+    "jest-watch-select-projects": "^1.0.0",
+    "kcd-scripts": "^4.1.0"
   },
   "eslintConfig": {
     "extends": "./node_modules/kcd-scripts/eslint.js",
     "rules": {
       "babel/no-invalid-this": "off",
-      "import/prefer-default-export": "off",
-      "import/no-unassigned-import": "off"
-    }
+      "import/no-unassigned-import": "off",
+      "import/prefer-default-export": "off"
+    },
+    "overrides": [
+      {
+        "files": ["**/__tests__/**/*.js"],
+        "rules": {
+          "import/no-extraneous-dependencies": "off"
+        }
+      }
+    ]
```

Comment:
IMO, this should be reverted and `"jsdom": "^16.0.1"` should be added to devDependencies.